init/fetch-linux-headers.sh

```diff
@@ -14,6 +14,9 @@ generate_headers()
   zcat /proc/config.gz > .config
   make ARCH=x86 oldconfig > /dev/null
   make ARCH=x86 prepare > /dev/null
+
+  # Clean up abundant non-header files to speed-up copying
+  find ${BUILD_DIR} -regex '.*\.c\|.*\.txt\|.*Makefile\|.*Build\|.*Kconfig' -type f -delete
```

Comment:
Probably I'd prefer something like `find -name "..." -type f ...`

Reply:
Unfortunately alpine uses busybox find so I was limited to using the regex flag

Reply:
Ah right! Nevermind.

Reply:
BTW, just thinking around at 4AM ... Tomorrow I'll try with something like
`ls | grep -E "..." | xargs rm`

Reply:
> 4AM

Get some sleep ;)